components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java

```diff
@@ -255,6 +263,8 @@ private IdentityApplicationManagementUtil() {
         samlAuthnContextComparisonLevelList.add(IdentityApplicationConstants.SAML2.AuthnContextComparison.MAXIMUM);
         samlAuthnContextComparisonLevelList.add(IdentityApplicationConstants.SAML2.AuthnContextComparison.BETTER);
         samlAuthnContextComparisonLevels = Collections.unmodifiableList(samlAuthnContextComparisonLevelList);
+
+        JS_LOOP_PATTERN = Pattern.compile("\\b(for|while|forEach)\\b");
```

Comment:
Let's move this outside the static block.

Reply:
done